### PR TITLE
dbaccessors

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by dannyroberts** [comment]
(Just waiting for build, already reviewed)

**Comment by dimagimon** [comment]
There are 5 errors:

* corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py, line 1 - F401 'get_all_case_owner_ids' imported but unused
* corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py, line 2 - F401 'get_closed_case_ids' imported but unused
* corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py, line 2 - F401 'get_open_case_ids' imported but unused
* corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py, line 3 - F401 'get_indexed_case_ids' imported but unused
* corehq/ex-submodules/casexml/apps/case/dbaccessors/__init__.py, line 3 - F401 'get_reverse_indexed_case_ids' imported but unused

**Comment by czue** [comment]
thanks for fixes!

**Comment by czue** [comment]
@dimagi/team-commcare-hq this is a model of how we're going to try and structure things that hit couch views (so that it's clear where they are and to one day get off them). You can see @dannyroberts's original rationale here: https://github.com/dimagi/commcare-hq/pull/6616#discussion_r29941765

For now just keep this in mind when making _new_ things, or feel free to move things over as you encounter them.

File: corehq/ex-submodules/casexml/apps/case/dbaccessors/by_owner.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    Get all open case ids for a given owner
+    """
+    return _get_case_ids(owner_id, False)
+
+
+def get_closed_case_ids(owner_id):
+    """
+    Get all closed case ids for a given owner
+    """
+    return _get_case_ids(owner_id, False)
```
Comment (NoahCarnahan) [inline corehq/ex-submodules/casexml/apps/case/dbaccessors/by_owner.py]: @dannyroberts Should you be passing `True` to `_get_case_ids` here?

Comment (czue) [inline corehq/ex-submodules/casexml/apps/case/dbaccessors/by_owner.py]: yes nice catch: https://github.com/dimagi/commcare-hq/pull/6624

Comment (czue) [inline corehq/ex-submodules/casexml/apps/case/dbaccessors/by_owner.py]: (clearly i need to add more tests for this)

**Pull request merged**